tests/Integration/Routing/RouteApiResourceTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Illuminate\Tests\Integration\Routing;
+
+use Orchestra\Testbench\TestCase;
+use Illuminate\Support\Facades\Route;
+use Illuminate\Tests\Integration\Routing\Fixtures\ApiResourceTaskController;
+use Illuminate\Tests\Integration\Routing\Fixtures\ApiResourceTestController;
+
+/**
+ * @group integration
+ */
+class RouteApiResourceTest extends TestCase
+{
+    public function test_api_resource()
+    {
+        Route::apiResource('tests', ApiResourceTestController::class);
+
+        $response = $this->get('/tests');
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertEquals('I`m index', $response->getContent());
+
+        $response = $this->post('/tests');
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertEquals('I`m store', $response->getContent());
+
+        $response = $this->get('/tests/1');
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertEquals('I`m show', $response->getContent());
+
+        $response = $this->put('/tests/1');
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertEquals('I`m update', $response->getContent());
+        $response = $this->patch('/tests/1');
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertEquals('I`m update', $response->getContent());
+
+        $response = $this->put('/tests/1');
```

Comment:
Duplicate put request?
```
$this->put
(...)
$this->patch
(...)
$this->put again
```
Looks like the same copy/paste issue twice in `test_api_resources`